nested-types.ipynb

```diff
@@ -0,0 +1,802 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Nested types\n",
+    "\n",
+    "* List, List View\n",
+    "* Struct\n",
+    "* Map\n",
+    "* Union\n",
+    "* Dictionary encoded,\n",
+    "* Run-end encoded\n",
+    "\n",
+    "We will have a look at first three types."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Fixed and variable size list\n",
+    "\n",
```

Comment:
I think we should start here with a brief explanation of what a "list" type is in general, before distinguishing the different types.

Reply:
I would maybe also start with only showing the first diagram for variable size list (which is the most common case, I think), then show the example code, and then show the diagram of fixed size list and again examples (so in the flow of the tutorial we can introduce fixed size as a kind of special case when all lists are the same size, the offsets are no longer needed)